### ISSUE TITLE
Unicode support

### ISSUE THREAD

**Issue by pbamotra** [TeamHG-Memex/eli5#233]
eli5.show_prediction leads to UnicodeEncodeError: 'ascii' codec can't encode character u'\ube54' in position 0: ordinal not in range(128)

Sorry can't share snapshots. Was trying on production data.

**Comment by kmike** [comment]
Can you show the whole traceback?

**Comment by pbamotra** [comment]
<img width="813" alt="screen shot 2017-08-04 at 11 37 42 am" src="https://user-images.githubusercontent.com/6505165/28982166-a6c9bd3e-7909-11e7-98d2-46eb730bcc14.png">

**Comment by chenjun0210** [comment]
I also encountered the same problem In Chinese

**Comment by kmike** [comment]
@chenjun0210 would it be possible to provide a reproducible example for this issue (or, even better, a pull request with a failing test case)?
